sdcm/sct_config.py

```diff
@@ -965,7 +965,13 @@ class SCTConfiguration(dict):
 
         # LongevityTest
         dict(name="stress_multiplier", env="SCT_STRESS_MULTIPLIER", type=int,
-             help=""),
+             help="Number of cassandra-stress processes"),
```

Comment:
If we are introducing a new SCT  parameter, we should generate the docs.

Also if they are relevant only to very specific cases, it should be documented